app/vep/utils/vcf_results.py

```diff
@@ -144,29 +146,70 @@ def _get_alt_allele_details(
     )
 
 
+def _get_meta(vcf_path: FilePath) -> model.VcfMetadata:
```

Comment:
If, as the comment says, this is a method for managing vcf file metadata, then perhaps its name should be `_get_vcf_file_meta`, or something of the sort?

Reply:
Good point, but I would instead rename Vcfmetadata to something else: this metadata (file) is also meant to contain other state info like currently selected filter query (not yet implemented).